tests/testthat/test-generate.R

```diff
@@ -1,27 +1,27 @@
-# context("generate")
-#
-# # gap_df <- gapminder %>%
-# #   filter(year == 2007 & continent %in% c("Africa")) %>%
-# #   specify(response = lifeExp)
-#
-# dummy_df <- data.frame(matrix(1:3, nrow = 3, ncol = 1))
-#
-# test_that("shape",{
-#
-#   # Testing shape output
-#   expect_equal(nrow(generate(x=dummy_df,n_samples=5)),15)
-#
-#   # If dataframe is empty
-#   expect_error(generate(x=data.frame(matrix(nrow=0,ncol=0))))
-#
-#   # If input integer for n_samples is negative
-#   expect_error(generate(x=gap_df,n_samples=-2))
-#
-#   # If number of samples chosen to be zero
-#   expect_silent(generate(x=gap_df,n_samples=0))
-#
-#   # If number of samples chosen is too high
-#   expect_error(generate(x=gap_df,n_samples=1e10000))
-#
-#   }
-#   )
+context("generate")
+
+gap_df <- gapminder::gapminder %>%
```

Comment:
If you have a chance tomorrow, during review, to change from the gapminder df. I loaded it last week because I wasn't sure, but the TA made a great point to not add unnecessary packages if possible. See if you can use iris, mtcars. But this is ok for now.